content/en/Platform Deep Dive/Cobalt Account/account-recovery.md

```diff
@@ -0,0 +1,119 @@
+
+Once you've set up a new authenticator, you can turn on 2FA again.<!--Add link to instruction-->
+
+#### Turn Off 2FA for a User
+
+As an [_Organization Owner_](/getting-started/glossary/#organization-owner), you can turn off two-factor authentication for a user following their request.
+
+1. Once you get an email notification requesting you to turn off 2FA, select **Recover Account** in the email.
+1. On the **People** page of your dashboard, locate the user who submitted the request.
+1. Select the three-dot icon on the right, and then select **Turn Off 2FA**.
+   - We verified the user's identity with a one-time verification code. However, we recommend that you verify their identity again before turning off 2FA.
+1. Select **Confirm** in the overlay that appears.
+
+### One-Time Codes Don't Work
+
+One-time codes that your authenticator app generates are time-based. If the codes that you enter when signing in are invalid, do the following:
+
+- Enter the code promptly, within 30 seconds as it appears in the authenticator app.
+- Make sure the time is synchronized between the devices. If you manually changed the time zone on your mobile device earlier, select the system time zone.
+
+### "Remember This Device" Doesn't Work
+
+When you sign in to Cobalt using 2FA, you can select **Remember this device for 30 days**. This option works for a specific device and browser.
+
+You need to enter a one-time code from your authenticator if:
+
+- You're using a different device or browser.
+- You cleared cookies from your browser, or you haven't enabled them.
+- You're browsing in incognito mode.
+- You're using a different internet connection.
+- Another user signed in on the same device using the same browser.
+
+## Forgot Your Password
+
+To reset your password:
+
+1. On the {{% sign-in %}} page, select **Forgot password?**.
+1. Enter your email address that you used to [sign in to Cobalt](/getting-started/sign-in/), and select **Reset Password**.
+1. Follow the instructions in the email you receive.
+
+## Can't Sign In Using SSO
+
+If you have problems signing in through [SAML SSO](/getting-started/sign-in/#saml-sso), contact your administrator for help. Ensure that:
+
+- You're signing in from the identity provider system, such as Okta or OneLogin, and not the Cobalt {{% sign-in %}} page.
+- You've completed the account setup process in the Cobalt app, which includes confirming your email address and password.
+
+## Locked Out of Your Account
+
+We'll temporarily lock your account in the following cases:
+
+- If you enter an incorrect password 10 times.
+- If you enter an invalid identity verification code 3 times when [recovering access without an authenticator app](#lost-access-to-your-authenticator).
+
+To unlock your account, follow the instructions in the email that you receive.
+
+Alternatively, try to sign in again after 5 minutes. If you still can't sign in, contact us at support@cobalt.io.
```

Comment:
> ## Locked Out of Your Account
> 
> We'll temporarily lock your account in the following cases:
> 
> - If you enter an incorrect password 10 times.
> - If you enter an invalid identity verification code 3 times when [recovering access without an authenticator app](#lost-access-to-your-authenticator).
> 
> To unlock your account, follow the instructions in the email that you receive.
> 
> Alternatively, try to sign in again after 5 minutes. If you still can't sign in, contact us at support@cobalt.io.

@molfinn Thank you for the info Molly. I faced a scenario when I can't sign in to several test accounts even if I enter a correct password after being locked out. Maybe it's related to using the same IP address and browser.

Since I ran out of test accounts, I'll follow your suggestion about 5 minutes and the number of unsuccessful attempts.

It goes beyond this question, but -- since you're already looped in -- can I ask you to look through this PR that is essentially reworked Zendesk content?

Preview: https://deploy-preview-225--cobalt-docs.netlify.app/platform-deep-dive/cobalt-account/account-recovery/

These sections were not changed since the 2FA recovery process was released:
- [Lost Access to Your Authenticator](https://deploy-preview-225--cobalt-docs.netlify.app/platform-deep-dive/cobalt-account/account-recovery/#lost-access-to-your-authenticator)
- [Forgot Your Password](https://deploy-preview-225--cobalt-docs.netlify.app/platform-deep-dive/cobalt-account/account-recovery/#forgot-your-password)

Reply:
@ana-dashuk-cobalt : i'd assume you were locked out at that time that you kept entering an incorrect password. Since you were using a test account you didn't receive an unlock instruction email. Random ? with this, do we need to get any sort of security buy in on the level of detail we're providing here? Could a hacker use this information. I know with some of our error messages we are purposefully vague so as to not give away too much info. 

My comments on the other preview are in the files.

Reply:
@molfinn Can we reword this into something more generic to keep the message safe?

> If you make several unsuccessful attempts to sign in, we'll temporarily lock your account.
> 
> To unlock your account, follow the instructions in the email that you receive. If you still can't sign in, contact us at [support@cobalt.io](mailto:support@cobalt.io).